index.html

```diff
@@ -241,6 +240,61 @@ <h3 class="about-head-text">Github</h3>
           <h3 class="about-head-text">Codepen</h3>
           <h3 class="about-head-text">Gitlab</h3>
           <h3 class="about-head-text">Terminal</h3>
+        </div>
+
+          <!-- contact me form -->
+
+          <footer>
+          <div class="contact-me">
+            <h1 class="contact-me-para">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h1>
```

Comment:
[Optional] - For semantic purposes and search engine optimization, kindly use just one h1 in an HTML file, other headings can be h2-h6, depending on their order of importance.